LEGAL.txt

```diff
@@ -1,3 +1,5 @@
+Original WarpX Only:
```

Comment:
I probably need to understand the workflow better, but generally speaking let us not modify this file.

If WarpX is used in a dependent, a good workflow could be to place it in a subdirectory.

Reply:
Sorry, yeah I missed this line. We're still somewhat figuring out which parts of our development can be merged back upstream and which not. We added a line on Modern Electron copyright to our internal version and I meant to remove it all, but missed that top line.

Reply:
No worries, thanks for the info! :)